Tests/TootSDKTests/PostTests.swift

```diff
@@ -0,0 +1,45 @@
+// Created by konstantin on 04/12/2022.
+// Copyright (c) 2022. All rights reserved.
+
+import XCTest
+
+@testable import TootSDK
+
+final class PostTests: XCTestCase {
+
+    func testPostEdited() throws {
+        // arrange
+        let json = localContent("post_edited")
+        let decoder = TootDecoder()
+
+        // act
+        let result = try decoder.decode(Post.self, from: json)
+
+        // assert
+        XCTAssertNotNil(result)
+        XCTAssertEqual(result.id, "112300188527054704")
+        XCTAssertEqual(result.filtered, [])
+        XCTAssertEqual(result.mentions, [])
+        XCTAssertEqual(result.emojis, [])
+        XCTAssertEqual(result.mediaAttachments, [])
+        XCTAssertEqual(result.tags.count, 1)
+        XCTAssertEqual(result.favourited, false)
+        XCTAssertEqual(result.reposted, false)
+        XCTAssertNotNil(result.createdAt)
+        XCTAssertNotNil(result.editedAt)
```

Comment:
Could you add an equality check as well, just to make sure the correct time and tz are parsed.

Reply:
Is there an example of a date check in the tests? I don't see any.

Reply:
We have some e.g. https://github.com/TootSDK/TootSDK/blob/main/Tests/TootSDKTests/ActivityTests.swift